pyproject.toml

```diff
@@ -72,6 +72,10 @@ testing = [
     "setuptools-scm", # Required for tests using python-ogr.spec
     "setuptools-scm-git-archive", # Required for tests using python-ogr.spec
 ]
+dev = [
+    "packitos[testing]",
```

Comment:
Isn't it possible to use just `.[testing]`?

Reply:
I have just tested, it doesn't work

Reply:
Ok, good to know.